client/app/components/compose.coffee

```diff
@@ -441,7 +441,7 @@ module.exports = Compose = React.createClass
             # to save temporary info
             unless @state.id
                 @state.id = message.id
-                Router.redirect
+                window.location.href = RouterGetter.getURL
```

Comment:
It is a temporary FIX; this should be moved into `action_creator` files, and use `navigate`from `Backbone.Router`